setup.py

```diff
@@ -13,7 +13,7 @@
     )
 
 
-with open('README.rst') as f:
+with open('README.rst', encoding="utf-8") as f:
```

Comment:
Please update to single quotes.
```suggestion
with open('README.rst', encoding='utf-8') as f:
```